ci/scripts/run-ecs-db-migration.js

```diff
@@ -93,7 +93,7 @@ const run = async function run () {
     console.log(`Current task definition is using release: ${currentAppRelease}`)
     const jobAppRelease = APPLICATION_IMAGE_TAG.split('-')[0]
 
-    if (currentAppRelease !== jobAppRelease) {
+    if (currentAppRelease !== jobAppRelease && CLUSTER_NAME != "test-perf-1-fargate") {
```

Comment:
Just a very minor thing but can you make it a !== so it is type checked too. I know it's bonkers but I do not trust JS even a little bit not to do some type shenanigans and make this be wrong :(